test/compareSnapshots-test.js

```diff
@@ -135,3 +141,20 @@ describe('when images are equal', () => {
     });
   });
 });
+
+describe('when fetchPng fails', () => {
+  beforeEach(() => {
+    let tries = 0;
+    fetchPng.mockImplementation((url) => {
+      if (tries > 1) {
+        return realFetchPng(url);
+      }
+      tries += 1;
+      return Promise.reject(new Error('What happened?'));
+    });
```

Comment:
I think this could maybe be simplified with https://jestjs.io/docs/en/mock-function-api#mockfnmockrejectedvalueoncevalue

```js
fetchPng
  .mockImplementation((url) => realFetchPng(url))
  .mockRejectedValueOnce(() => new Error('What happened?'));
```

Reply:
You're right. I might have some time later to address, but for now I'm leaving as-is.